test/IISExpress.FunctionalTests/InProcess/FixtureLoggedTest.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    public class FixtureLoggedTest: LoggedTest
+    {
+        private readonly IISTestSiteFixture _fixture;
+
+        public FixtureLoggedTest(IISTestSiteFixture fixture)
```

Comment:
Why not make the rest of the fixture tests inherit FixtureLoggedTest?

Reply:
I'd like to ease it in. See if it breaks things